### ISSUE TITLE
Ruby interpreter override not working in Mass deployment

### ISSUE THREAD

**Issue by OnixGH** [phusion/passenger#1818]
Given the config:
```
app1/Passengerfile.json:
{
  "port": 3001,
  "ruby": "/usr/local/rvm/gems/ruby-2.2.4/wrappers/ruby"
}

app2/Passengerfile.json:
{
  "port": 3002,
  "ruby": "/usr/local/rvm/gems/ruby-2.3.0/wrappers/ruby"
}

(puts RUBY_VERSION in app1 and app2 config.ru)
```

..and running Passenger Enterprise 5.0.28 (standalone) from the base dir, the Ruby version that gets printed is the version that Passenger was started with, rather than what is specified in the configs.

If running Passenger standalone directly from the app dir, the override does work.

**Comment by benflex** [comment]
This is great @OnixGH, thanks a lot. When should this fix (and 5.0.29 I suppose) be released ?

**Comment by OnixGH** [comment]
The best estimate I have at the moment is somewhere in June, after https://github.com/phusion/passenger/issues/1807 and hopefully https://github.com/phusion/passenger/issues/1794 are taken care of.

In the meantime, it's a small patch that you could apply manually (no Passenger recompilation needed either).
